core/common/src/main/java/alluxio/security/authentication/SaslStreamClientDriver.java

```diff
@@ -72,10 +77,11 @@ public void onNext(SaslMessage saslMessage) {
       LOG.debug("SaslClientDriver received message: {}",
           saslMessage != null ? saslMessage.getMessageType().toString() : "<NULL>");
       SaslMessage response = mSaslHandshakeClientHandler.handleSaslMessage(saslMessage);
-      if (response == null) {
-        mRequestObserver.onCompleted();
```

Comment:
`onCompleted` is not required anymore?

Reply:
onCompleted() calls for both client and server are reserved now for long polling. Calling them would indicate that the end of authenticated session. 
When called from client, it means "I don't want to be authenticated anymore". 
When called from server, it means, for the client, that "I'm not authenticated anymore."

Reply:
I see. Would it be useful to have this information somewhere in the comments? I'm just not sure where is the best place to put this info.